src/networking.c

```diff
@@ -539,6 +539,119 @@ get_mac_address(const gchar *const ifname)
 	return macaddr;
 }
 
+/*
+ * Get and store the iptable rules set in the
+ * network namespace. These need to be re-applied
+ * within the container.
+ *
+ * \param config \ref cc_oci_config
+ *
+ * \return \c true on success, else \c false.
+ */
+gboolean
+get_iptable_rules(struct cc_oci_config *config)
+{
+	gboolean ret;
+	GError *error = NULL;
+	gint exit_status;
+	char *args[2] = { "iptables-save", NULL };
```

Comment:
Why not just drop the `2` and let the compiler deal with it? Also, `const` maybe?